oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -614,9 +640,12 @@ impl Runtime {
         &self,
         node_id: NodeId,
         readers: &[Handle],
-    ) -> Result<Vec<ChannelReadStatus>, OakStatus> {
-        self.validate_handles_access(node_id, readers)?;
-        self.validate_can_read_from_channels(node_id, readers)?;
+    ) -> Result<Vec<ChannelReadStatus>, ErroneousChannelStatus> {
```

Comment:
Presumably there's a core invariant here, that the length of any returned `Vec` has the same length as the `readers` input, i.e. for both:
 - `Ok(statuses)` (pre-existing)
 - `Err(ErroneousChannelStatus::HasInvalid(statuses))` (new).

Please could we document that clearly in the function comment?